src/cli/config/config.go

```diff
@@ -178,13 +180,21 @@ func Read(ctx context.Context) error {
 
 // WriteRepoConfig currently just persists corso config to the config file
 // It does not check for conflicts or existing data.
-func WriteRepoConfig(ctx context.Context, s3Config storage.S3Config, m365Config account.M365Config) error {
-	return writeRepoConfigWithViper(GetViper(ctx), s3Config, m365Config)
+func WriteRepoConfig(ctx context.Context,
```

Comment:
Style standards: the first parameter should take its own line, not stay inlined.
```suggestion
func WriteRepoConfig(
    ctx context.Context,
```